### PR TITLE
Create the Google Cloud Platform node functionalities

### PR DESCRIPTION
**Pull request by yitzhak12**
Create the Google Cloud Platform node functionalities. 
Fix issue https://issues.redhat.com/browse/OCSQE-1494.

### CODE REVIEW

File: ocs_ci/utility/gcp.py [hunk 1]
```diff
@@ -109,3 +112,157 @@ def compute_driver(self):
...
+    project_id = None
+
+    def __init__(self, project_id=None):
+        """
+        Constructor for GoogleCloud class
+
+        """
+        os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = SERVICE_ACCOUNT_KEY_FILEPATH
+        self.compute = build("compute", "v1", cache_discovery=False)
+        self.project_id = project_id or "odf-qe"
```
Comment (ebenahar) [inline ocs_ci/utility/gcp.py:168]: please save "odf-qe" as a constant

**Review by ebenahar — COMMENTED** [review]
Great work Itzhak

File: ocs_ci/utility/gcp.py (hunk 1 above)
Comment (yitzhak12) [reply ocs_ci/utility/gcp.py:168]: Yes, np.

**Review by ebenahar — APPROVED** [review]

**Comment by yitzhak12** [comment]
There are more failures in the current GCP tier1 run:https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/31280/testReport/ than in the previous GCP tier1 run: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/9585/testReport/, but overall the failures look similar.

**Review by OdedViner — APPROVED** [review]

**Comment by yitzhak12** [comment]
The tier4a node-related tests passed here: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/31090/testReport/. 
About the tier4b failed tests: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-test-pr/6038/testReport/ - I analyzed the failures and saw that it's related to general automation issue and general issues unrelated to GCP specific.

**Pull request merged**